internal/rcache/rcache.go

```diff
@@ -172,32 +171,45 @@ func (r *Cache) Increase(key string) {
 	}
 }
 
+// DeleteMulti deletes the given keys.
+func (r *Cache) DeleteMulti(keys []string) {
```

Comment:
nit: `GetMulti` uses varargs, so we might want to keep consistency here with varargs as well:

```suggestion
func (r *Cache) DeleteMulti(keys ...string) {
```